.gitignore

```diff
@@ -46,3 +46,4 @@ __pycache__
 
 # Ignore echidna artifacts
 crytic-export
+.vscode/settings.json
```

Comment:
This file is not ignored by upstream: https://github.com/ethereum-optimism/optimism/blob/develop/.vscode/settings.json

Reply:
reverted.